.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Release
+env:
+    EXODUS_API_KEY: ${{secrets.EXODUS_API_KEY}}
+on:
+  push:
+    tags:
+      - release-v*
+      
+  workflow_dispatch:
+jobs:
+    release:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout the code ✅
```

Comment:
**suggestion**: If we want to add icons, can we put it in the main workflow as well?